core/java/test/org/openda/exchange/dataobjects/GenericNetcdfDataObjectTest.java

```diff
@@ -0,0 +1,125 @@
+package org.openda.exchange.dataobjects;
+
+import junit.framework.TestCase;
+import org.openda.interfaces.IExchangeItem;
+import org.openda.utils.OpenDaTestSupport;
+
+import java.io.File;
+
+public class GenericNetcdfDataObjectTest extends TestCase {
+	private OpenDaTestSupport testData;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'testData' could be replaced by a local variable.](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=4902599)